presto-docs/src/main/sphinx/presto_cpp/properties.rst

```diff
@@ -277,3 +277,38 @@ The configuration properties of Presto C++ workers are described here, in alphab
 * **Default value:** ``0.25``
 
   See description for ``shared-arbitrator.memory-pool-min-free-capacity``
+
+Memory Checker Properties
+-----------------------------
```

Comment:
Please align the `---` with the previous line.

Reply:
Done